### PR TITLE
add api util for sending raw transactions

### PR DESCRIPTION
**Pull request by alecananian**
- Adds a new API utility for the `POST /transactions/raw` endpoint that allows consumers to queue up raw transactions (either with pre-created data or for sending native tokens)
- Renames the Identity tab of the example app to Session
- Adds new examples for minting MAGIC and sending ETH to the example app

### CODE REVIEW

File: Assets/Treasure/Example/Scripts/IdentityUI.cs [hunk 1]
```diff
@@ -45,4 +47,73 @@ public async void OnAuthTokenSubmit(string value)
...
+        {
+            TDKLogger.LogError($"Mint failed: {transaction.errorMessage}");
+        }
+        else
+        {
+            TDKLogger.LogInfo($"Mint successful: {transaction.transactionHash}");
+        }
+    }
+
+    public async void OnRawMintMagicBtn()
```
Comment (lefarchi) [inline Assets/Treasure/Example/Scripts/IdentityUI.cs:74]: Is there a difference between doing it "raw" here vs the above? or is it just as an usage example of the new `SendRawTransaction` function?

**Review by lefarchi — APPROVED** [review]
Nice, these buttons are useful to have in the harness

File: Assets/Treasure/Example/Scripts/IdentityUI.cs [hunk 2]
```diff
@@ -1,4 +1,6 @@
 using System;
```
Comment (lefarchi) [inline Assets/Treasure/Example/Scripts/IdentityUI.cs:1]: Should we rename this file to SessionUI?

File: Assets/Treasure/Example/Scripts/IdentityUI.cs (hunk 1 above)
Comment (alecananian) [reply Assets/Treasure/Example/Scripts/IdentityUI.cs:74]: Just a usage example

File: Assets/Treasure/Example/Scripts/IdentityUI.cs (hunk 2 above)
Comment (alecananian) [reply Assets/Treasure/Example/Scripts/IdentityUI.cs:1]: Just going to keep it as-is for now to minimize file and scene changes 🫡

**Pull request merged**